modules/peers.js

```diff
@@ -292,7 +371,7 @@ Peers.prototype.acceptable = function (peers) {
 	return _.chain(peers).filter(function (peer) {
 		// Removing peers with private or host's ip address
 		return !(ip.isPrivate(peer.ip) || ip.address('public', 'ipv4', true).some(function (address) {
-			return [address, library.config.port].join(':') === [peer.ip, peer.port].join(':');
+			return (address + ':' + library.config.port) === (peer.ip + ':' + peer.port);
```

Comment:
Can we stick to joining array for now? This standard set throughout. We can change all separately if deemed better.